### PR TITLE
[AND-3034] MoPub 6.8.0 RC build chores

### PR DESCRIPTION
**Pull request by anton-blashchuk**
https://vungle.atlassian.net/browse/AND-3034

### CODE REVIEW

**Review by alexander-gubov — DISMISSED** [review]
LGTM, but aars changes need to be reverted

File: Vungle/CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
 ## Changelog
+  * 6.8.0.0
+    * This version of the adapters has been certified with Vungle 6.8.0 and MoPub SDK 5.13.0.
```
Comment (alexander-gubov) [inline Vungle/CHANGELOG.md:3]: better to have the removal of withFlexViewCloseTimeInSec API mentioned here.

**Review by alexander-gubov — DISMISSED** [review]

**Review by HuFangshuai — DISMISSED** [review]
LGTM

**Review by HuFangshuai — DISMISSED** [review]
LGTM, since PR approved we can submit upstream PR then.

**Review by HuFangshuai — APPROVED** [review]
LGTM.

**Review by alexander-gubov — APPROVED** [review]

**Pull request merged**